tests/jax/mocks.py

```diff
@@ -834,3 +839,15 @@ def design(self) -> Tuple[DesignSpec, Dict]:
             return DesignSpec(**components), {}
 
     return TestSystem()
+
+
+@dataclass
+class MockTimeTerminatorConfig(TimeTerminatorConfig):
+    termination_function: Callable = lp.stop
+    run_seconds: float = 1.0
+
+
+@dataclass
+class MockCountConditionTerminatorConfig(CountConditionTerminatorConfig):
+    termination_condition: Optional[Dict[str, Any]] = None
+    termination_function: Callable = lp.stop
```

Comment:
I could be wrong here, but I don't think these need to be mocks, you should just be able to use the original configs. My understanding is that you only mock out something that has undesired functionality, but these are simply datastores, so they have no functionality and I don't think you need to mock them out

Reply:
Totally agree. Sorted.